app/models/empty_search_response.rb

```diff
@@ -0,0 +1,17 @@
+class EmptySearchResponse
```

Comment:
This class is exists to show the initial state of that page. Making it inherit from `SearchResponse` wouldn't make us write less code since we would need to override all the methods anyway (apart from the last one). By keeping it separate there is less code to browse when looking around. Does that make sense @jackscotti ?